code/modules/xenoarcheaology/effect.dm

```diff
@@ -118,29 +118,10 @@
 	if(!istype(H))
 		return 1
 
-	var/protected = 0
+	. = 1
+	for(var/obj/item/I in H.get_equipped_items())
+		var/datum/extension/armor/armor = get_extension(I, /datum/extension/armor)
+		if(armor)
+			. -= armor.get_value("anomaly")/100
```

Comment:
I mean, should at least run it through `get_blocked` or whatever, and allow the key to be fetched. Defeats the point of armor datums like this.

Reply:
made it item var instead